src/helper.js

```diff
@@ -50,6 +50,13 @@ const parseBlindingKey = (swap) => {
     return swap.blindingKey ? Buffer.from(swap.blindingKey, "hex") : undefined;
 };
 
+export const cropString = (str) => {
+    if (str.length < 40) {
+        return str;
+    }
+    return str.substring(0, 20) + "..." + str.substring(str.length - 20);
```

Comment:
```suggestion
    return str.substring(0, 19) + "..." + str.substring(str.length - 19);
```

Else it won't fit in one line

![image](https://github.com/BoltzExchange/boltz-web-app/assets/8138838/9d6eecfa-1f67-4a41-9203-f856804a234e)